.circleci/config.yml

```diff
@@ -99,6 +99,8 @@ jobs:
       - checkout
       - *restore_cache
       - run: |
+          rm -rf node-modules
+          npm install
```

Comment:
I guess you had no luck on fixing this? I might see if I can figure out how to get CI to pass without this since this feels not great

Reply:
Got it. Some of the dependencies aren't being cached between CircleCI steps properly. A fix for this is [here](https://github.com/mattermost/mattermost-webapp/commit/b4eca4cf3426a970c077dfdfc05cde7751f67c8a)